_data/simple-icons.json

```
@@ -5492,6 +5492,11 @@
             "hex": "246FDB",
             "source": "https://tagmanager.google.com/#/home"
         },
+        {
+            "title": "Google Tasks",
+            "hex": "2684FC",
+            "source": "https://assistant.google.com/tasks"
```

And do we need to add the usual Google guidelines?
```suggestion
            "source": "https://assistant.google.com/tasks",
            "guidelines": "https://about.google/brand-resource-center"
```